packages/flutter_tools/lib/src/device.dart

```diff
@@ -63,14 +63,14 @@ class DeviceManager {
         device.name.toLowerCase().startsWith(deviceId);
 
     final Device exactMatch = await devices.firstWhere(
```

Comment:
you should be able to remove this await since its a list and no longer a stream now?